### ISSUE TITLE
Wrong loss on discriminator

### ISSUE THREAD

**Issue by manurare** [leftthomas/SRGAN#46]
Hey! I found that when optimizing the discriminator its loss is defined as:
`d_loss = 1-real_out+fake_out`
which translates into D(G(z))+1-D(x). Shouldn't it be:
`d_loss = 1+real_out-fake_out` 
so it matches the discriminator definition D(x)+1-D(G(z))?

**Comment by leftthomas** [comment]
@manurare no, we want real_out to be close 1, and fake_out to close 0
